Objects/rangeobject.c

```diff
@@ -138,6 +131,28 @@ range_new(PyTypeObject *type, PyObject *args, PyObject *kw)
     return NULL;
 }
 
+static PyObject *
+range_new(PyTypeObject *type, PyObject *args, PyObject *kw)
+{
+    if (!_PyArg_NoKeywords("range", kw))
+        return NULL;
+
+    return range_from_array(type, _PyTuple_ITEMS(args), PyTuple_GET_SIZE(args));
+}
+
+
+static PyObject *
+range_vectorcall(PyTypeObject *type, PyObject *const *args,
+                 size_t nargsf, PyObject *kwnames)
+{
+    size_t nargs = PyVectorcall_NARGS(nargsf);
```

Comment:
Oh, I didn't expect a size_t here. I opened an issue to discuss PyVectorcall_NARGS() return type: https://bugs.python.org/issue39611

Reply:
The return type already is Py_ssize_t. The bug is only in this patch.
(When Mark was originally writing the patch, there was a discussion of `Py_ssize_t` vs. `size_t`. Mark is a big proponent of the unsigned type, which *is* more correct. But also much less compatible with the existing codebase.)